tools/cloud-workstations/Dockerfile

```diff
@@ -31,6 +31,7 @@ RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add -  && \
     curl https://packages.cloud.google.com/apt/doc/apt-key.gpg \
       | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add - && \
     apt-get -y update && apt-get -y install google-cloud-sdk && \
+    apt-get -y install tmux && \
```

Comment:
just add `tmux` to line 28 at the end list of other packages

Reply:
Done